docs/configuring-polaris-for-production.md

```diff
@@ -55,6 +55,13 @@ Be sure to secure your metastore backend since it will be storing credentials an
 
 To use [EclipseLink](https://eclipse.dev/eclipselink/) for metastore management, specify the configuration `metaStoreManager.conf-file` to point to an [EclipseLink `persistence.xml` file](https://eclipse.dev/eclipselink/documentation/2.5/solutions/testingjpa002.htm). This file, local to the Polaris service, will contain information on what database to use for metastore management and how to connect to it.
 
+> [!IMPORTANT]
+> To use [EclipseLink](https://eclipse.dev/eclipselink), you need to ensure that you have done two things:
+> 1. Build the jar for the EclipseLink extension
+> 2. Set the property on the Polaris project to include eclipseLink.
+>
+> This can be achieved by setting `eclipseLink=true` in the `gradle.properties` file, or by passing the property explicitly while building all jars, e.g.: `./gradlew -PeclipseLink=true clean assemble`
+
```

Comment:
I think we need something like this for sure. But I don't think this is really connected to _configuring_ Polaris, but building it. Maybe there's a better place for this, or a new doc we need to write?

Reply:
That makes sense. I added it here because this is where I found myself looking for more information. Do you have any pointers/ideas on where it would fit better?

Reply:
I think it's okay to have it here, approved for now with a few comments. In the future it might be neat to have a doc enumerating the different MetastoreManager options, and we could link to that doc from here.

Reply:
Thanks @eric-maynard. I've made the changes as per your comments.